app/frontend/js/modals/createModal.js

```diff
@@ -0,0 +1,11 @@
+import { Modal } from "bootstrap";
+
+const createModal = document.getElementById("create-modal");
+
+if (createModal) {
```

Comment:
This code looks very similar to addHarvestModal.js.  Does codeclimate pick it up?  You could consolidate it

Reply:
Yeah that sounds good :). I actually haven't run codeclimate for a bit :(